src/style.css

```diff
@@ -2,0 +1,114 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf) format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  src: url(./fonts/Avenir-Heavy.ttf) format('truetype');
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
Be consistent with your margins. You've added only a top margin here, which is good, but ensure that this approach is consistent across your CSS to avoid potential margin collapse issues.